projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -1135,6 +1137,17 @@ function ( $category_id ) {
 
 					break;
 
+				case 'in_site_migration_flow':
+					$canonical_value = (int) (bool) $value;
+					if ( ! (bool) $value ) {
```

Comment:
non blocker, but it might be clearer to check this way: `if ( 0 === $canonical_value )`